src/views/MainView.js

```diff
@@ -36,15 +36,20 @@ class MainView extends PureComponent {
   }
 
   shouldShowOverlay = () => {
+    const html = document.documentElement;
     const now = new Date();
     const currentTimeInHours = now.getHours();
     
     const shouldShowOverlay =
       currentTimeInHours < dayStartTimeInHours || currentTimeInHours >= dayEndTimeInHours;
 
-      return shouldShowOverlay;
+    if (shouldShowOverlay && html && !html.classList.contains('overlay-is-active')) {
+      html.classList.add('overlay-is-active');
+    };
```

Comment:
this should remove `overlay-is-active` when shouldShowOverlay is false.

It should be more like:

```
if (shouldShowOverlay) {
  // ensure `overlay-is-active` is on the document
} else {
  // ensure `overlay-is-active` is removed from the document
}
```